hack/generate-manifest.sh

```diff
@@ -401,7 +418,9 @@ if $K8S_115; then
 fi
 
 $KUSTOMIZE build
-
+if $SIMULATOR; then
+    cat ../../patches/simulator/antrea-agent-simulator.yml
+fi
```

Comment:
you should use `$KUSTOMIZE edit add resource antrea-agent-simulator.yml` instead, in the previous `if $SIMULATOR` block

Reply:
done